pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -20,19 +20,31 @@
 package virt_controller
 
 import (
+	"fmt"
+
 	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"kubevirt.io/client-go/log"
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 
 	k6tv1 "kubevirt.io/api/core/v1"
 )
 
 var (
 	vmStatsCollector = operatormetrics.Collector{
-		Metrics:         append(timestampMetrics, vmResourceRequests, vmInfo),
+		Metrics:         append(timestampMetrics, vmResourceRequests, vmInfo, vmDiskAllocatedSize),
 		CollectCallback: vmStatsCollectorCallback,
 	}
 
+	vmDiskAllocatedSize = operatormetrics.NewGaugeVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vm_disk_allocated_size_bytes",
+			Help: "Allocated disk size in bytes for each Virtual Machine.",
```

Comment:
@avlitman please check the how the pvc name label is called also in other metrics, to make sure we are aligned.